### ISSUE TITLE
Pytorch: autograd example

### ISSUE THREAD

**Issue by xuyuji9000** [xuyuji9000/machine-learning-playground#37]
[Automatic Differentiation with torch.autograd](https://pytorch.org/tutorials/beginner/basics/autogradqs_tutorial.html)

**Comment by xuyuji9000** [comment]
Pytorch gradient descent example added.

Closing this issue.
